### PR TITLE
Update windows-dev-env.md

### PR DESCRIPTION
**Pull request by memory125**
Update the windows-dev-env.md on how to set up the IoT development environment for Windows 10 platform.

### CODE REVIEW

**Comment by memory125** [comment]
Update the windows-dev-env.md on how to set up the IoT development environment for Windows 10 platform.

File: doc/machine-config/windows-dev-env.md [hunk 1]
```diff
@@ -86,6 +91,15 @@ Configuration steps, scripts and tools i use on windows machines. Feel free to s
   pushd "%USERPROFILE%\Development"
   ```
 
+  If you installed Visual Studio 2015 and put the following,
```
Comment (seank-com) [inline doc/machine-config/windows-dev-env.md:94]: Nt: remove and

File: doc/machine-config/windows-dev-env.md [hunk 2]
```diff
@@ -75,8 +79,9 @@ Configuration steps, scripts and tools i use on windows machines. Feel free to s
   atom Bin\Init.cmd Bin\cmds.lst
   ```
 
-  For Init.cmd put the following
+  For Init.cmd.
 
+  If you installed Visual Studio 2013 and put the following,
```
Comment (seank-com) [inline doc/machine-config/windows-dev-env.md:84]: Nit:remove and

File: doc/machine-config/windows-dev-env.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 Configuration steps, scripts and tools i use on windows machines. Feel free to skip any steps that don't work for you or add steps if you think something is missing.
 
+### Install Visual Studio, Azure SDK, etc
+
+Please install Visual Studio, Azure SDK, etc by this [here](https://github.com/Azure/azure-iot-sdks/blob/master/csharp/device/doc/devbox_setup.md).
```
Comment (seank-com) [inline doc/machine-config/windows-dev-env.md:7]: Nit: (I generally prefix my comments with Nit: to indicate that I am being Nitpicky and I'm just correcting grammar or style. You should feel free to ignore any of my comments prefixed with Nit:)

, etc by this [here]

might sound better this way

, etc using the steps [here]

**Comment by seank-com** [comment]
👍  You should look for a thumbs up from your code reviewer before merging into develop

**Pull request merged**